subsys/bluetooth/audio/Kconfig.mediacontrol

```diff
@@ -0,0 +1,50 @@
+# Bluetooth Audio - Media control configuration options
+
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BT_AUDIO
+
+config BT_MEDIACONTROL
```

Comment:
```suggestion
config BT_MEDIA_CONTROL
```

Reply:
I left the underscore out on purpose, to try to indicate that this was one level.  Do you think  it is two?
(Or don't you think it matters?)

Reply:
Hmm, I actually never considered additional underscores in Kconfig to add another "level", but I sure see how it could be seen that way. I am not familiar enough with any traditions we have for that in Zephyr.